app/controllers/ApprovalController.scala

```diff
@@ -31,7 +32,8 @@ class ApprovalController @Inject() (approvalService: ApprovalService, cc: Contro
   def save: Action[JsValue] = Action.async(parse.json) { implicit request =>
     val process = request.body.as[JsObject]
 
-    approvalService.save(process).map {
+    // This will change when we have additional end point for fact check
+    approvalService.save(process, REVIEW_TYPE_2I, STATUS_SUBMITTED_FOR_2I_REVIEW).map {
```

Comment:
Looks like Java :), Scala standard for constants is upper camel case , see https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods. Could add them to a package object

Reply:
Ok, it def looks like java :) I had a look at the enums we have already and seemed OTT for what these are needed for.  I'll follow the naming conventions you mentioned